### PR TITLE
Document person put_userinfo action

### PR DESCRIPTION
**Pull request by danidoni**
Document `person#put_userinfo` action.

You can test it in [our review app](https://obs-reviewlab.opensuse.org/danidoni-document-person-put-userinfo-action/apidocs-new/)

-----------------

If this PR requires any particular action or consideration before deployment,
please check the reasons or add your own to the list:

* [ ] Requires a database migration that can cause downtime. Postpone deployment until maintenance window[1].
* [ ] Contains a data migration that can cause temporary inconsistency, so should be run at a specific point of time.
* [ ] Changes some configuration files (e.g. options.yml), so the changes have to be applied manually in the reference server.
* [ ] A new Feature Toggle[2] should be enabled in the reference server.
* [ ] Proper documentation or announcement has to be published upfront since the introduced changes can confuse the users.

[1] https://github.com/openSUSE/open-build-service/wiki/Deployment-of-build.opensuse.org#when-there-are-migrations
[2] https://github.com/openSUSE/open-build-service/wiki/Feature-Toggles-%28Flipper%29#you-want-real-people-to-test-your-feature

### CODE REVIEW

**Comment by danidoni** [comment]
[Person](https://trello.com/c/LrjRFGxI/1596-person)

File: src/api/public/apidocs-new/components/responses/person/not_found.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+description: |
+  Not found.
+
+  XML Schema used for body validation: [status.xsd](../schema/status.xsd)
```
Comment (saraycp) [inline src/api/public/apidocs-new/components/responses/person/not_found.yaml:4]:
```suggestion
  XML Schema used for body validation: [status.xsd](../../schema/status.xsd)
```

Comment (danidoni) [reply src/api/public/apidocs-new/components/responses/person/not_found.yaml:4]: Ouch, thanks!

Comment (danidoni) [reply src/api/public/apidocs-new/components/responses/person/not_found.yaml:4]: Talked offline and we agreed on disregarding this suggestion.

File: src/api/public/apidocs-new/paths/person_login.yaml [hunk 2]
```diff
@@ -1,24 +1,16 @@
-post:
-  summary: Perform changes on a registered person
+put:
+  summary: Update existing person details
```
Comment (dmarcoux) [inline src/api/public/apidocs-new/paths/person_login.yaml:2]:
```suggestion
  summary: Update person
```

File: src/api/public/apidocs-new/paths/person_login.yaml [hunk 3]
```diff
@@ -1,24 +1,16 @@
...
-      schema:
-        type: string
-        enum: [change_password,lock,delete]
-      required: true
-      description: "The command to execute against the provided person."
-      example: change_password
   requestBody:
-    description: "The password in plain text when the cmd param is 'change_password'."
-    required: false
+    description: The data to update for the person specified in the login parameter.
```
Comment (dmarcoux) [inline src/api/public/apidocs-new/paths/person_login.yaml:8]: This sentence is a bit weird. Would this be better?

```suggestion
    description: The new data for the person referred in the login parameter.
```

Comment (danidoni) [reply src/api/public/apidocs-new/paths/person_login.yaml:8]: Yes, much better. Thanks!

**Comment by dmarcoux** [comment]
I cannot comment on that line in the file _paths/person_login.yaml_ because it wasn't changed, but it's broken. The link to the `api_response.xsd` doesn't work. Should it be `api_response.yaml` instead?

**Comment by danidoni** [comment]
> I cannot comment on that line in the file _paths/person_login.yaml_ because it wasn't changed, but it's broken. The link to the `api_response.xsd` doesn't work. Should it be `api_response.yaml` instead?

No, this will 

> I cannot comment on that line in the file _paths/person_login.yaml_ because it wasn't changed, but it's broken. The link to the `api_response.xsd` doesn't work. Should it be `api_response.yaml` instead?

I can't find any `api_response.xsd` 🤔 ...

**Comment by saraycp** [comment]
> > I cannot comment on that line in the file _paths/person_login.yaml_ because it wasn't changed, but it's broken. The link to the `api_response.xsd` doesn't work. Should it be `api_response.yaml` instead?
> 
> I can't find any `api_response.xsd` thinking ...

What @dmarcoux comments is not introduced in this PR, it was part of the post_userinfo one.

https://github.com/openSUSE/open-build-service/blob/a9401a6fc6f3b4ad8af3969c138d710b9986d132/src/api/public/apidocs-new/paths/person_login.yaml#L27

**Comment by dmarcoux** [comment]
@saraycp Yes, you are right. I got confused while reviewing both PRs one after the other.

**Review by dmarcoux — DISMISSED** [review]

**Comment by danidoni** [comment]
Yes, I'll create a new example for the 404 because for this endpoint it's something different than the one from POST

**Review by saraycp — APPROVED** [review]
LGTM

**Pull request merged**